css/index.css

```diff
@@ -1,3 +1,328 @@
+}
 
+@media(max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+}
 
+/* Header Styles */
+
+header {
+    width: 100%;
+    display: flex;
+}
+
+@media(max-width: 800px) {
+    header {
+        flex-wrap: wrap;
+    }
+}
+
+@media(max-width: 500px) {
+    header {
+        justify-content: center;
+    }
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+    display: flex;
+}
+
+@media(max-width: 800px) {
+    header nav {
+        width: 768px;
+        padding: 0 30px;
+
+    }
+}
+
+@media(max-width: 500px) {
+    header nav {
+        width: 100%;
+        flex-direction: column;
+        margin: 0;
+        padding: 0;
+    }
+}
+
+header nav a {
+    text-decoration: none;
+    color: #b0b0b0;
+}
+
+@media(max-width: 500px) {
+    header nav a {
+        width: 100%;
+        display: block;
+        text-align: center;
+        padding: 20px 0;
+        font-size: 20px;
+        border-bottom: 1px solid gray;
+    }
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+@media(max-width: 500px) {
+    header nav a:hover {
+        background: gray;
+    }
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+@media(max-width: 500px) {
+    header .logo {
+        margin: 20px 0;
+    }
+}
+
+@media(max-width: 800px) {
+    header .logo {
+        order: -1;
+        display: block;
+        margin: 30px auto 0;
+    }
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+@media(max-width: 800px) {
+    .cta {
+        justify-content: center;
+    }
+}
+
+@media(max-width: 500px) {
+    .cta {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center
+    }
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+@media(max-width: 800px) {
+    .cta .cta-text {
+        margin: 0;
+    }
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 16px;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+@media(max-width: 800px) {
+    .cta img {
+        display: none;
+    }
+}
+
+@media(max-width: 500px) {
+    .cta img {
+        max-width: 80%;
+    }
+}
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+}
+
+@media(max-width: 800px) {
+    .main-content {
+        margin: 75px 20px 35px;
+    }
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+@media(max-width: 500px) {
+    .main-content.top-content {
+        justify-content: space-around;
```

Comment:
you have the correct properties here to do what you want, but you need to add a space in between .main-content.top-content to get them to work here